lib/resty/radixtree.lua

```diff
@@ -210,6 +212,140 @@ local function insert_tab_in_order(tab, val, func)
     insert_tab(tab, val)
 end
 
+local function update_tab_in_order(tab, val, func)
+    local idx = -1
+    for i, j in ipairs(tab) do
+        if func(val, j) then
+            move_tab(tab, i, #tab, i + 1)
+            tab[i] = val
+
+            if idx == -1 then
+                for x = i + 1, #tab do
+                    if tab[x].id == val.id then
+                        remove_tab(tab, x)
+                        break
+                    end
+                end
+            else
+                remove_tab(tab, idx)
+            end
+
+            return
+        end
+
+        if j.id == val.id then
+            idx = i
+        end
+    end
+    insert_tab(tab, val)
+    remove_tab(tab, idx)
+end
+
+local function modify_route(self, opts)
```

Comment:
The functions needs to be separated by two blank lines, please check other similar places

Reply:
ok. I will put two blank lines between functions.